packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -58,16 +58,23 @@ function InteractionLayer ({
       return true
     }
 
-    const activeMark = activeTool.createMark({
+    const activeMark = activeTool.onPointerDown({
```

Comment:
I think `createMark` should remain, but shouldn’t be called if `creating` is true. If we are already in the middle of creating a mark, then we delegate the pointerDown event to the active mark, which can then handle it if necessary.

If `creating` is false, then we call `activeTool.createMark()` and start a new mark.

Reply:
So maybe leave the original mark creation code in place, but put this before it. 

```js
if (creating) {
  return activeMark.handlePointerDown(event)
}
```

Reply:
I like the idea of the active tool handling events during mark creation. That removes the ambiguity between tapping to place a transcription line and tapping on a transcription line to select and move it.

Reply:
I like the suggestions here. It's a bit more clear and keeps the mark creation decoupled from handling pointer down. I'd still suggest doing something like #1736 and have a generic handler in the base `Tool` model. I'll close it for now, but I'll leave the branch as a reference.